pkg/innerring/processors/netmap/process_peers.go

```diff
@@ -116,3 +154,60 @@ func (np *Processor) processUpdatePeer(ev netmapEvent.UpdatePeer) {
 		np.log.Error("can't invoke netmap.UpdatePeer", zap.Error(err))
 	}
 }
+
+var errUnknownDifference = errors.New("announced and received information differ but not critical")
+
+func compareNodeInfos(niExp, niGot netmap.NodeInfo) error {
+	// if a node bootstraps the first time it is OK
+	// to be OFFLINE but send ONLINE bootstrap requests
+	niGot.SetOnline()
```

Comment:
`if niGot.IsOffline()`?

Reply:
> states will be almost definitely different

do not agree, health check pings should be the majority

> maybe SetOnline() for both instances?

agree that it should allow maintenance state transfer, fixed